.changeset/ninety-numbers-vanish.md

```diff
@@ -0,0 +1,15 @@
+---
+"@itwin/itwinui-css": major
+---
+
+Add `.iui-field` class to unify & centralize basic styling for `.iui-button`, `.iui-input`, `.iui-input-flex-container`, and `.iui-select-button`.
+
+- Paginator buttons now use the `.iui-field` data attribute for sizing rather than the `.iui-table-paginator-page-button-small` modifier class.
+- `.iui-select-tag-button` now requires `.iui-button .iui-field` and `data-iui-variant="borderless"`.
+- `.iui-breadcrumbs-content` now requires `.iui-button .iui-field`.
+- `.iui-sidenav-button .iui-expand` now requires `.iui-button .iui-field` and `data-iui-size="small"`.
+- Feedback button type now requires `data-iui-variant`.
```

Comment:
Should this be the ideas button?

```suggestion
- Ideas button type now requires `data-iui-variant`.
```

Also, not sure if this point is outdated. Since we now need to add `.iui-button-idea` + `data-iui-variant="high-visibility"` instead of `data-iui-variant="idea"`?

Reply:
Corrected, thanks!